modules/provision/main.tf

```diff
@@ -284,9 +150,7 @@ resource "aws_iam_role" "vespa_cloud_provisioner_role" {
 
 resource "aws_iam_policy" "vespa_cloud_provision_policy" {
   name   = "vespa-cloud-provisioner-policy"
-  policy = (var.is_cd ?
-     data.aws_iam_policy_document.provision_policy_cd.json :
-     data.aws_iam_policy_document.provision_policy.json)
+  policy = data.aws_iam_policy_document.provision_policy.json
```

Comment:
But this encourages rollout to both system at the same time.

Reply:
We should split this up if we roll out some experimental changes again. This PR only adds a single permission, so it should be safe.